rush.json

```diff
@@ -282,25 +282,25 @@
       "projectFolder": "misk-web/web/packages/@misk/tslint",
       "versionPolicyName": "MiskWeb"
     },
-    {
-      "packageName": "misk-web-examples-data",
-      "projectFolder": "examples/data",
-      "versionPolicyName": "MiskWebDemo"
-    },
-    {
-      "packageName": "misk-web-tab-dashboard",
-      "projectFolder": "examples/services/dashboard",
-      "versionPolicyName": "MiskWebDemo"
-    },
+    // {

```

Comment:
can we remove these?

Reply:
Planning to add them back in a future PR. They were becoming too difficult to fix and aren't a priority for landing the exemplar code.

Reply:
okay, in the future it'll be helpful to at least leave a comment on plans for other readers. like:
`// TODO(adrw): Add back in at a later time as it's not needed for exemplar code`